projects/aca-content/src/lib/components/search/search-results-row/search-results-row.component.ts

```diff
@@ -86,13 +94,42 @@ export class SearchResultsRowComponent implements OnInit, OnDestroy {
     this.node = this.context.row.node;
     this.isFile = this.node.entry.isFile;
 
-    const { name, properties } = this.node.entry;
-    const title = properties ? properties['cm:title'] : '';
-
+    const highlights: SearchEntryHighlight[] = this.node.entry['search']?.['highlight'];
```

Comment:
Maybe it would be good idea to update model to have search and highlight fields defined inside models? Thanks that we won't need to do some workarounds with [] and IDE will hint available fields. 

Not sure if that is easy to update and if that is possible so if not in that PR we can also raise task for that. What do you think?

Reply:
Basically `ResultSetRowEntry` contains search properties but the component itself needs regular node to pass it to subsequent methods accepting node only, and node itself shouldn't contain the search properties imo @DenysVuika what do you think?